internal/contrib/telemetrytest/telemetry_test.go

```diff
@@ -70,7 +72,7 @@ func testTelemetryEnabled(t *testing.T, contribPath string) error {
 	if err = os.Chdir(contribPath); err != nil {
 		return err
 	}
-	body, err := exec.Command("go", "list", "-json", "./...").Output()
+	body, err := exec.Command("go", "list", "-json", "."+separator+"...").Output()
```

Comment:
I think `go list` and all the Go toolchain uses `/` for referring relative packages.